Makefile

```diff
@@ -16,29 +23,96 @@ osrm:
 phpunit:
 	@docker compose exec php php bin/console doctrine:schema:update --env=test --force --no-interaction --quiet
 	@docker compose exec php php vendor/bin/phpunit
+# Add as annotation in any testcase:
+# /**
+# * @group only
+# */
+# public function testSomething()..
+phpunit-only:
+	@clear && docker compose exec php php vendor/bin/phpunit --group only
 
 behat:
 	@docker compose exec php php vendor/bin/behat
+# For now, just change here the `features/file.feature:xx` to run a specific test
+behat-only:
+	@clear && docker compose exec php php vendor/bin/behat features/stores.feature:96
```

Comment:
It's also possible to add a `@debug` tag on top of a scenario, and then run `vendor/bin/behat --tags="@debug"`

Reply:
Niceeee, done, thanks!